### PR TITLE
Show escaped version of keyboard emulator server return

### PR DESCRIPTION
**Pull request by asdil12**
Log lines will look like this:
```
[2022-11-29T14:44:04.481416+01:00] [debug] Keyboard emulator says: "type: root\n"
```

VR: https://openqa.suse.de/tests/10050269

### CODE REVIEW

**Review by perlpunk — APPROVED** [review]

**Review by Martchus — APPROVED** [review]
I'll approve but next time move this duplicated code into a separate function.

**Review by perlpunk — CHANGES_REQUESTED** [review]
Oops, there is a typo in the title and commit message

**Comment by asdil12** [comment]
> Oops, there is a typo in the title and commit message

Fixed

**Review by okurz — CHANGES_REQUESTED** [review]
I suggest to un-duplicate the code. Three lines of identical code is enough to warrant a small refactoring

**Comment by asdil12** [comment]
> I suggest to un-duplicate the code. Three lines of identical code is enough to warrant a small refactoring

you asked for it ;)

File: consoles/video_stream.pm [hunk 1]
```diff
@@ -235,29 +235,43 @@ sub send_key_event ($self, $key, $press_release_delay) {
...
+    chomp($server_response);
+    bmwqemu::diag("Keyboard emulator says: " . bmwqemu::pp($server_response));
+}
+
+
+sub type_string ($self, $args) {
+    if ($bmwqemu::vars{GENERAL_HW_KEYBOARD_URL}) {
+        $self->keyboard_emulator_cmd(type => $args->{text});
         return {};
     }
```
Comment (Martchus) [inline consoles/video_stream.pm:268]: The new function could also return `{}`, then this code and the other occurrence boil down to:
```suggestion
    return $self->keyboard_emulator_cmd(type => $args->{text}) if $bmwqemu::vars{GENERAL_HW_KEYBOARD_URL};
```

**Review by perlpunk — APPROVED** [review]

**Review by Martchus — APPROVED** [review]

**Pull request merged**